readme.md

```diff
@@ -18,6 +18,10 @@ isUrl('https://sindresorhus.com');
 
 isUrl('unicorn');
 //=> false
+
+// Enable lenient mode to automatically prepend `https://` to the input in case it has no protocol defined.
+isUrl('unicorn', {lenient: true});
+//=> true
```

Comment:
Needs to be properly documented. Example: https://github.com/sindresorhus/boxen#api

Reply:
Readme updated. Now it's similar to https://github.com/sindresorhus/string-length#api, which has only one boolean option.